controller/ingress_entry.go

```diff
@@ -40,6 +40,10 @@ type IngressEntry struct {
 	CreationTimestamp time.Time
 	// Ingress resource
 	Ingress *v1beta1.Ingress
+	//
```

Comment:
`golint` complains for a reason - exported properties should have a simple explanation.